tests/data/simple_cases/function_trailing_comma.py

```diff
@@ -1,5 +1,11 @@
 def f(a,):
     d = {'key': 'value',}
+    e = {
```

Comment:
Can you add similar test cases using set, tuple, and list literals?

Reply:
Thanks for the comment. Yes, I've just added those.